### PR TITLE
Create info.ini

### PR DESCRIPTION
**Pull request by Sneezry**

### CODE REVIEW

**Comment by jimaek** [comment]
Can you add minified files to your repo and use those in these configs?

**Comment by Sneezry** [comment]
Done :-)

File: files/gridjs/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "gridjs",
+  "repo": "gridjs/gridjs",
+  "files": {
+    "include": ["gridjs.js", "gridjs.min.js"]
```
Comment (jimaek) [inline files/gridjs/update.json]: Great, remove `gridjs.js` from here and its good to go.

Comment (Sneezry) [inline files/gridjs/update.json]: Done, thank you for your advising :-)

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**